### PR TITLE
AM-11500: Fixed the typos in v1.0.0 and v1.1.0 register cluster docs

### PR DESCRIPTION
**Pull request by uma-kt**

### CODE REVIEW

File: versioned_docs/version-1.0.0/install-kubeslice/yaml/yaml-register-worker-clusters.mdx [hunk 1]
```diff
@@ -200,7 +195,7 @@ file to install the Slice Operator. By completing these steps, the worker cluste
 integrate with the KubeSlice Controller and utilize its features.
 
 :::info
-** THIS STEP NEEDS TO BE PERFORMED ON ALL WORKER CLUSTERS IN THE CONFIGURATION. **
+This step needs to be performed on all worker clusters in the configuration.
```
Comment (veenamj-avesha) [inline versioned_docs/version-1.0.0/install-kubeslice/yaml/yaml-register-worker-clusters.mdx:198]: Let this be in upper case for more visibility.

File: versioned_docs/version-1.0.0/install-kubeslice/yaml/yaml-register-worker-clusters.mdx [hunk 2]
```diff
@@ -488,11 +485,12 @@ This parameter is related to network operations such as enforcing the QoS profil
...
 Use base64 encoded values of the `namespace`, `endpoint`, `ca.crt`, and
-the `token` from the above secrets retrieved by using the `secret.sh` script or the output following the manual secret retrieval steps to create your `sliceoperator.yaml` file.
+the `token` from the above secrets retrieved by using the `secret.sh` script or the output following the manual secret retrieval
+steps to create the `sliceoperator.yaml` file.
 
-### Create YAML template
+### Create the YAML File
 
-Copy and pase the below template into a file called `sliceoperator.yaml`
+Copy and pase the below template into a file called `sliceoperator.yaml`.
```
Comment (veenamj-avesha) [inline versioned_docs/version-1.0.0/install-kubeslice/yaml/yaml-register-worker-clusters.mdx:493]: copy and paste

File: versioned_docs/version-1.0.0/install-kubeslice/yaml/yaml-register-worker-clusters.mdx [hunk 3]
```diff
@@ -523,7 +521,7 @@ https://34.159.173.186
 ```
 
 
-### Example of the Slice Operator YAML
+The following is the example of the Slice Operator YAML:
```
Comment (veenamj-avesha) [inline versioned_docs/version-1.0.0/install-kubeslice/yaml/yaml-register-worker-clusters.mdx:524]: Let this be a heading

**Review by veenamj-avesha — COMMENTED** [review]
Left a few comments on version 1.0.0 that also applies to version 1.1.0. Can you address all of them?

File: versioned_docs/version-1.0.0/install-kubeslice/yaml/yaml-register-worker-clusters.mdx (hunk 1 above)
Comment (uma-kt) [reply versioned_docs/version-1.0.0/install-kubeslice/yaml/yaml-register-worker-clusters.mdx:198]: done

File: versioned_docs/version-1.0.0/install-kubeslice/yaml/yaml-register-worker-clusters.mdx (hunk 2 above)
Comment (uma-kt) [reply versioned_docs/version-1.0.0/install-kubeslice/yaml/yaml-register-worker-clusters.mdx:493]: done

**Review by veenamj-avesha — APPROVED** [review]
Changes LGTM.

**Pull request merged**